### ISSUE TITLE
Keyboard crashes when trying to open and close it multiple times. 

### ISSUE THREAD

**Issue by paritosh9199** [florisboard/florisboard#1208]
#### Environment information
- FlorisBoard Version: 0.3.12 (43)
- Device: OnePlus LE2101
- Android: 11 (cn=R sdk=30)

#### Features enabled
```kt
smartbar = true
suggestions = true
suggestions_clipboard = true
suggestions_next_word = true
glide = true
clipboard_internal = true
clipboard_history = true
```

#### Attached stacktrace files
<details>
<summary>1630268911447.stacktrace</summary>

```
java.lang.NullPointerException
	at dev.patrickgold.florisboard.ime.core.FlorisBoard$Companion.getInstance(FlorisBoard.kt:150)
	at dev.patrickgold.florisboard.ime.text.TextInputManager.getFlorisboard(TextInputManager.kt:63)
	at dev.patrickgold.florisboard.ime.text.TextInputManager.getActiveEditorInstance(TextInputManager.kt:67)
	at dev.patrickgold.florisboard.ime.text.TextInputManager.access$getActiveEditorInstance(TextInputManager.kt:59)
	at dev.patrickgold.florisboard.ime.text.TextInputManager$onUpdateSelection$2$1.invokeSuspend(TextInputManager.kt:459)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)

```
</details>

**Comment by Glitchy-Tozier** [comment]
Possibly related to #1206.

**Comment by paritosh9199** [comment]
Closing this as this seems to be fixed in 0.3.13
